dali_tf_plugin/dali_tf_plugin_utils.py

```diff
@@ -49,7 +49,14 @@ def get_tf_compiler_version():
     lib = tensorflow_libs[0]
     cmd = 'strings -a ' + lib + ' | grep "GCC: ("'
     s = str(subprocess.check_output(cmd, shell=True))
-    version = re.search("GCC:\s*\(.*\)\s*(\d+.\d+).\d+", s).group(1)
+    lines = s.split('\\n')
+    version = ''
+    for line in lines:
+        res = re.search("GCC:\s*\(.*\)\s*(\d+.\d+).\d+", line)
+        if res:
+            ver = res.group(1)
+            if not version or version < ver:
+                version = ver
```

Comment:
I think that `group` returns `string` or `tuple`. In that case `<` is alphabetical order, I think. Maybe explicit `int(ver)` would be better here?

Reply:
done